modules/ROOT/nav.adoc

```diff
@@ -18,6 +18,7 @@
 .Advanced Data Operations
 * xref:howtos:concurrent-async-apis.adoc[Async APIs]
 * xref:howtos:concurrent-document-mutations.adoc[Concurrent Document Mutations]
+* xref:howtos:encrypting-using-sdk.adoc[Encrypting Your Data]
 * xref:howtos:transcoders-nonjson.adoc[Transcoders & Non-JSON]
 //* xref:howtos:durability.adoc[Durability]
```

Comment:
realise this is commented out but just noticed it's out of order :D

Reply:
Probably an old page that isn't used anymore? Not sure, but leaving out of this change.